AK/FlyString.h

```diff
@@ -51,8 +51,9 @@ class FlyString {
     // This is primarily interesting to unit tests.
     [[nodiscard]] static size_t number_of_fly_strings();
 
-    // FIXME: Remove this once all code has been ported to FlyString
+    // FIXME: Remove these once all code has been ported to FlyString
     [[nodiscard]] DeprecatedFlyString to_deprecated_fly_string() const;
+    static ErrorOr<FlyString> from_deprecated_string(DeprecatedFlyString const&);
```

Comment:
Shouldn't this be `from_deprecated_fly_string()` to mirror the API right above it?

Reply:
It could. I called it `from_deprecated_string()` since it can take both `DeprecatedString` and `DeprecatedFlyString` and didn't want to mislead anyone thinking it requires a `DeprecatedFlyString`.

I can change it to `from_deprecated_fly_string()` if we still want to.

Reply:
Updated per Ali's suggestion